### PR TITLE
feat: update feishu notification template

### PR DESCRIPTION
**Pull request by NinoFocus**
⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]:
- [x] I have read and understand the pull request rules.

# Description

Fixes #(issue)

## Type of change

Please delete any options that are not relevant.

- User interface (UI)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [x] I have commented my code, particularly in hard-to-understand areas (including JSDoc for methods)
- [x] My changes generates no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

## Screenshots (if any)

Please do not use any external image service. Instead, just paste in or drag and drop the image here, and it will be uploaded automatically.

| Event | Before | After |
|-------|-------|------|
| `UP` | image | image |
| `DOWN` | image | image |

### CODE REVIEW

File: server/notification-providers/feishu.js [hunk 1]
```diff
@@ -25,51 +25,59 @@ class Feishu extends NotificationProvider {
...
-                                                "[Down] " +
-                                                heartbeatJSON["msg"] +
-                                                `\nTime (${heartbeatJSON["timezone"]}): ${heartbeatJSON["localDateTime"]}`
-                                        },
-                                    ],
-                                ],
+                    msg_type: "interactive",
+                    card: {
+                        config: {
+                            update_multi: true,
```
Comment (CommanderStorm) [inline server/notification-providers/feishu.js:31]: Sorry, but the translation via google translate is not really making sense on this parameter.
Could you give context what this does?

image

**Review by CommanderStorm — COMMENTED** [review]
Overall, this looks good to me.
I am a bit unsure what `update_multi` does..

File: server/notification-providers/feishu.js (hunk 1 above)
Comment (NinoFocus) [reply server/notification-providers/feishu.js:31]: Sorry, my mistake, update_multi should be set to `false`

Comment (NinoFocus) [reply server/notification-providers/feishu.js]: This parameter is used to update the content of the message card sent by the application. In the monitor scenario, this is not necessary.

**Comment by CommanderStorm** [comment]
Thanks for the change to the notification provider! 🎉 



> [!NOTE]
> This PR is part of the [`v2.0` merge window](https://github.com/louislam/uptime-kuma/milestone/24) => see #4500 for the bugs that need to be addressed before we can ship this release ^^

<sub>All changes in this PR are small and uncontroversial ⇒ merging with junior maintainer approval</sub>

**Pull request merged**